modules/microsoft.cdn/azure-frontdoor-cdn-profile/1.0/azuredeploy.parameters.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
remove param values when you add default values to the template

Reply:
Fixed.